### DISCUSSION TITLE
Screen shake

### DISCUSSION THREAD

**Discussion by Stripes09** [ramokz/phantom-camera#407] [Question / Help Wanted] [answered]
I have been trying to implement screen shake in the phantom camera, but I cant get the camera to move outside its designated position, how can I achieve screen shake?

**Comment by Stripes09** [comment, marked as answer]
Never mind I love you so much, the noise effect is beutiful and you deserve applause for it, this really is the best addon Ive used Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You Thank You.

For anyone wondering just in case, you just have to go to the nose property and set the amplitide up when you need to, in code that is
(whateveryourphantomcamnodeis).noise.amplitude = (number)

**Reply by ramokz** [reply]
You're very welcome :smile:
